### PR TITLE
Partition promtail role

### PR DESCRIPTION
**Pull request by thanche1**

### CODE REVIEW

File: partition/roles/promtail/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# TODO
```
Comment (robertvolkmann) [inline partition/roles/promtail/README.md:1]: Still empty.

File: partition/roles/promtail/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+# TODO: move to release vector
```
Comment (robertvolkmann) [inline partition/roles/promtail/defaults/main.yml:2]:
```suggestion
```

File: partition/roles/promtail/templates/promtail.yaml.j2 [hunk 3]
```diff
@@ -0,0 +1,74 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /var/log/promtail-positions.yaml
+
+clients:
+  - url: {{ promtail_loki_push_endpoint }}/loki/api/v1/push
```
Comment (robertvolkmann) [inline partition/roles/promtail/templates/promtail.yaml.j2:9]:
```suggestion
  - url: {{ promtail_loki_push_endpoint }}
```
`/loki/api/v1/push` should be part of `promtail_loki_push_endpoint`.

File: partition/roles/promtail/templates/promtail.yaml.j2 [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  #     level:
+  #     component:
+  #     machineID:
+  # - metrics:
+  #     log_lines_total:
+  #       type: Counter
+  #       description: "total number of log lines"
+  #       source: component
+  #       config:
+  #         action: inc
```
Comment (robertvolkmann) [inline partition/roles/promtail/templates/promtail.yaml.j2:41]: Do we need the above lines?

File: defaults/main.yaml [hunk 5]
```diff
@@ -90,15 +90,11 @@ metal_stack_release:
...
-    logging_chart_repo: "helm-charts.logging.loki-stack.repository"
+    logging_chart_version: "helm-charts.logging.loki.version"
+    logging_chart_repo: "helm-charts.logging.loki.repository"
     prometheus_chart_version: "helm-charts.monitoring.kube-prometheus-stack.version"
     prometheus_stack_repo: "helm-charts.monitoring.kube-prometheus-stack.repository"
     # monitoring
-    rethinkdb_exporter_name: "docker-images.metal-stack.control-plane.rethinkdb-exporter.name"
-    rethinkdb_exporter_tag: "docker-images.metal-stack.control-plane.rethinkdb-exporter.tag"
-    event_exporter_name: "docker-images.third-party.control-plane.event-exporter.name"
-    event_exporter_tag: "docker-images.third-party.control-plane.event-exporter.tag"
```
Comment (robertvolkmann) [inline defaults/main.yaml:101]: These exporters are still in use.

File: control-plane/roles/logging/templates/loki_ingress.yaml [hunk 6]
```diff
@@ -1,27 +1,26 @@
...
+    nginx.ingress.kubernetes.io/auth-secret: loki-basic-auth
+{% endif %}
 {% if logging_ingress_loki_tls %}
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
-{% if logging_ingress_loki_client_verify_ca %}
-    nginx.ingress.kubernetes.io/auth-tls-verify-client: "on"
-    nginx.ingress.kubernetes.io/auth-tls-secret: "{{ logging_namespace }}/loki-ca-secret"
-    nginx.ingress.kubernetes.io/auth-tls-verify-depth: "1"
-{% endif %}
```
Comment (robertvolkmann) [inline control-plane/roles/logging/templates/loki_ingress.yaml:24]: Why we're switched from client certificate to basic authentication?

**Review by robertvolkmann — CHANGES_REQUESTED** [review]

File: partition/roles/monitoring/ipmi-exporter/templates/config.yml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+modules:
```
Comment (robertvolkmann) [inline partition/roles/monitoring/ipmi-exporter/templates/config.yml:1]: How is using this file?

**Review by robertvolkmann — CHANGES_REQUESTED** [review]
Should promtail be moved to the monitoring directory?

**Review by robertvolkmann — APPROVED** [review]

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**